mixin/alerts.libsonnet

```diff
@@ -0,0 +1,104 @@
+{
+  new(this): {
+    name: 'SnowflakeAlerts',
+    rules: [
+      {
+        alert: 'SnowflakeWarnHighLoginFailures',
+        expr: |||
+          100 * sum by (job, instance) (last_over_time(snowflake_failed_login_rate{}[1h])) / sum by (job, instance) (last_over_time(snowflake_login_rate{}[30m]))
```

Comment:
Why are these two metrics using separate time ranges? e.g. `[1h]` and `[30m]`

Reply:
This was mostly just copied over from the original. Yep good catch 👍